### ISSUE TITLE
Accessibility audit: page turning status

### ISSUE THREAD

**Issue by cbeer** [ProjectMirador/mirador#3123]
> Once a digitised item is open, users can navigate through the pages/views of the digitised material using the arrow buttons "Previous item" and "Next item/Next item". While sighted users perceive the "turning of pages" directly, blind users are not informed that the view has changed.
> To change this, a live region could be created using the role="status" attribute. After paging, the text (e.g., the text below the digitized page number and name) should be placed inside the element with the role="status" attribute. This would mean that the text would be read by the screen reader after turning the page without shifting the focus.

**Comment by cbeer** [comment]
I'm not clear what text is expected; "5 of 10"? the new canvas label?

**Comment by jvine** [comment]
Clarified by Google Translate:

> To change this, a live region could be created using the attribute role = "status". After turning the page, **the text below the digitized material (page number and description)** should be placed within the element with the attribute role = "status".

A new region and role don't seem to be necessary; the problem is effectively solved by adding `aria-live="polite"` to the `<section>` that contains the `<canvas>`. 

`<section class="mirador-osd-container mirador236" id="window-3e034757-96f0-40c1-bf79-3d541b9cf4af-osd" aria-label="Item: Vol I (5) Barcelona. Eciia" aria-live="polite">

However there is duplication between the `aria-label` on this `<section>` and the requirements of #3107. It looks like this will resolve that - or in any case, it works in my test on VO:

- remove `aria-label="text"` from this `<section>`
- replace with `aria-labelledby="id-of-mirador-osd-info"` (ID established in #3107)
